iree/compiler/Conversion/LinalgToLLVM/MaterializeCPULaunchConfigurationPass.cpp

```diff
@@ -131,6 +132,8 @@ void MaterializeCPULaunchConfigurationPass::runOnOperation() {
       OwningRewritePatternList canonicalization(&getContext());
       AffineMinOp::getCanonicalizationPatterns(canonicalization, context);
       populateAffineMinSCFCanonicalizationPattern(canonicalization);
+      IREE::Flow::DispatchTensorLoadOp::getCanonicalizationPatterns(
```

Comment:
this is a brittle - it would be nice to have a populateFlowDispatchCanonicalizationPatterns in FlowOps.h so that as new ops that end up in dispatch regions get introduced they don't have to be specified in each file (specifically, constant loading, store, and the workgroup information ops).

Reply:
thanks!